classes/class-p4-post-campaign.php

```diff
@@ -29,24 +29,25 @@ public function __construct() {
 		 * Class hooks.
 		 */
 		private function hooks() {
-
 			add_action( 'init', [ $this, 'register_campaigns_cpt' ] );
+			add_action( 'cmb2_admin_init', [ $this, 'register_campaigns_metaboxes' ] );
 			add_action( 'add_meta_boxes', [ $this, 'campaign_page_templates_meta_box' ] );
+			add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_admin_assets' ] );
 			add_action( 'save_post_campaigns', [ $this, 'save_campaign_page_templates_meta_box_data' ] );
```

Comment:
You should change this hook `save_post_campaign`, since post type name has been changed